Assignment 5 Nicholas Burlakov Dillon Pikulik

NicholasBurlakov opened the pull request:
https://ambitious-grass-04916f41e.3.azurestaticapps.net/
Nicholas Burlakov and Dillon Pikulik
Several things do not work. Adding duplicate words is still allowed, MasterOfTheUniverse is not implemented nor is apriori. Other than that we have a solid start!

GrantErickson committed:
Auth Tables

GrantErickson committed:
Tokens and other auth

GrantErickson committed:
Update README.md

GrantErickson committed:
Updates to Auth on client side.

meghanmae committed:
Update Final.md

GrantErickson committed:
Minor class updates.

GrantErickson committed:
Update Lectures.md

GrantErickson committed:
Merge pull request #122 from IntelliTect-Samples/Auth

Auth merge into Main

GrantErickson committed:
Add testing to API action

GrantErickson committed:
Route guards

GrantErickson committed:
Class work

GrantErickson committed:
Cleanup

NicholasBurlakov committed:
Update azure-app-service-api.yml

NicholasBurlakov committed:
created word editor view

NicholasBurlakov committed:
Merge remote-tracking branch 'origin/Assignment-5' into Assignment-5

NicholasBurlakov committed:
update background theme

NicholasBurlakov committed:
prettier

NicholasBurlakov committed:
WordList
-shows words in wordlist
-change common flag of word list
UpdateWordList
-add word
-delete word
* I looked at bfristad's repo

NicholasBurlakov committed:
prettier

NicholasBurlakov committed:
Api edits to manipulate word list

NicholasBurlakov committed:
WordFlag
-changes common flag of word (removed from WordList)

NicholasBurlakov committed:
prettier

NicholasBurlakov committed:
More API edits

VForev committed:
Update Wordle.Api.sln test delete

VForev committed:
test delete

VForev committed:
prog back...

VForev committed:
Update azure-app-service-api.yml

NicholasBurlakov committed:
trying to get word editor page to respond (it doesnt)

NicholasBurlakov committed:
Merge remote-tracking branch 'origin/Assignment-5' into Assignment-5

NicholasBurlakov committed:
potential fix

NicholasBurlakov committed:
fixed. manipulating word list now avalible

NicholasBurlakov committed:
fixed about page bug, moved word editor to its own view

NicholasBurlakov committed:
aesthetic changes

VForev committed:
....

VForev committed:
fix api

NicholasBurlakov committed:
Configuring api/trying to implement Ruler of the universe

NicholasBurlakov committed:
Merge remote-tracking branch 'origin/Assignment-5' into Assignment-5

NicholasBurlakov committed:
prettier

NicholasBurlakov committed:
Api broken?!

NicholasBurlakov committed:
it needs this for authenticating but it breaks the api?!?

schemcharger commented on wordle-web/src/views/LoginView.vue:
logging in with incorrect credentials routes me to the main page.

schemcharger reviewed (commented):
Create an editor for the available words ✅
Create a page named "WordEditor" that allows words to be edited ✅
Support deleting a word ✅
Support adding a new word ✅
Editing words is not supported ✅
Duplicate words should not be allowed ❌
Support changing the common word boolean flag ❌✅ - I can't tell if it is working
Words should be sorted alphabetically ❌
Provide a text box for searching words AS YOU TYPE which supports searching for words starting with letters specified ❌
Provide a menu item for navigating to the WordEditor page ✅
Anyone can look at the word list ✅
Word list is paginated (10-100 per page) ✅
Any logged in user can change the common word flag ✅
There is a login page/component/popup/etc for users to sign in ✅
Only users over 21 years of age (based on birthday) with a claim of MasterOfTheUniverse can add and remove words ❌ - I could seemingly add words when not logged in
Users can be set up at start up in the API project as a seed (apriori), there does not need to be a sign up mechanism ✅

VForev committed:
appying fixs

NicholasBurlakov committed:
start here

NicholasBurlakov committed:
possible axios fix

NicholasBurlakov committed:
pret

pret

VForev committed:
....

VForev committed:
Merge remote-tracking branch 'origin/Assignment-5' into Assignment-5

VForev committed:
Dillon Reop droped on Nicks

The website is now running on dillons azure, we have a few things missing but are now moving focus to the final project.

VForev committed:
Nick db connections

VForev commented:
If the link to the website is bugging out, refresh the page a few times, I don't know why but eventually it works. But if you can't get it at all then you can use this one, this is a link to my repo site which is the same thing just running on my Azure. 
[https://white-flower-0e3bf9b1e.3.azurestaticapps.net/](url)

meghanmae commented on Wordle.Api/Wordle.Api/Controllers/WordController.cs:
```diff
@@ -35,6 +36,13 @@ public async Task<Word> AddWord(string newWord, bool isCommon)
             return await _wordService.AddWordAsync(newWord, isCommon);
         }
 
+        [HttpDelete("Delete")]
```
You need auth policies on these
